lib/models/rules.js

```diff
@@ -29,18 +29,23 @@ var util = require('util'),
     myutils = require('../myutils'),
     noSignal = require('./noSignal'),
     actions = require('./actions'),
+    namePattern = /^[a-zA-Z0-9_-]+$/,
+    MaxNameLength = 50,
```

Comment:
I understand that this PR enforces a name lengtha and a valid characters set. This should be included in the documentation (corresponding .md file, not sure which one should be the best one right now...).